### DISCUSSION TITLE
how can i connect to ethereum p2p network without node?

### DISCUSSION THREAD

**Discussion by BrazilRaw** [paradigmxyz/reth#4283] [Help] [answered]
Is it possible to connect to an Ethereum P2P network without running a node, and if so, how can one accomplish this?

**Comment by mattsse** [comment, marked as answer]
see 

https://github.com/paradigmxyz/reth/blob/main/examples/network.rs

https://github.com/paradigmxyz/reth/blob/main/examples/network-txpool.rs

**Reply by BrazilRaw** [reply]
Amazing ! Thank you so much !
